src/Adapter/DbSelect.php

```diff
@@ -122,7 +122,9 @@ public function count()
         $result    = $statement->execute();
         $row       = $result->current();
 
-        $this->rowCount = (int) $row[self::ROW_COUNT_COLUMN_NAME];
+        $this->rowCount = isset($row[self::ROW_COUNT_COLUMN_NAME])
+            ? (int) $row[self::ROW_COUNT_COLUMN_NAME]
+            : (int) $row[strtolower(self::ROW_COUNT_COLUMN_NAME)];
```

Comment:
What happens if this element is not present in the array?

Reply:
previously, it uses array in fields, so it added column to array